unreleased/fix-broken-link.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
Can you remove this file? I'll add a label to skip the changelog validation. I don't think this change should be part of the changelog.

Reply:
@jpkrohling  file removed.